src/registry.js

```diff
@@ -108,6 +108,18 @@ exports.register = function register(_modules) {
     }
 };
 
+exports.getUpdateOnPanComponents = function() {
```

Comment:
I was imagining a stashed list (similar to [these](https://github.com/wayne-o/plotly.js/blob/1479e9fd9880717659f8ee571202e5aba01d9626/src/registry.js#L23-L33)), not a getter method. This way we won't have to loop over all the registered components at the fps drag rate.